Assets/MixedRealityToolkit-Examples/UX/Scripts/DialogExampleLaunchButton.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MixedRealityToolkit.InputModule.EventData;
+using MixedRealityToolkit.UX.Dialog;
+using MixedRealityToolkit.UX.Buttons;
+using System.Collections;
+using UnityEngine;
+
+namespace MixedRealityToolkit.Examples.UX
+{
+    public class DialogExampleLaunchButton : MonoBehaviour
+    {
+        [SerializeField]
+        private Dialog dialogPrefab;
+
+        [SerializeField]
+        private bool isDialogLaunched;
+
+        [SerializeField]
+        private GameObject resultText;
+
+        [SerializeField]
+        private int numButtons = 1;
+
+        public GameObject ResultText
+        {
+            get
+            {
+                return resultText;
+            }
+
+            set
+            {
+                resultText = value;
+            }
+        }
+
+        public int NumButtons { get; set; }
```

Comment:
Does this need to be set in the inspector?
If so we may want to also have a private serialized backing field.